plugins/com.python.pydev.analysis/src/com/python/pydev/codecompletion/ctxinsensitive/CtxParticipant.java

```diff
@@ -218,7 +218,7 @@ private void fillNatureCompletionsForConsole(IScriptConsoleViewer viewer, int re
             FastStringBuffer displayString = new FastStringBuffer();
             FastStringBuffer tempBuf = new FastStringBuffer();
 
-            boolean doIgnoreImportsStartingWithUnder = AnalysisPreferences.doIgnoreImportsStartingWithUnder();
```

Comment:
You can get the project here as:

`request.nature != null ? request.nature.getProject(): null`

Reply:
Thanks, I'll do that.